playbooks/group_vars/all/13-dropbear_initrfs.yaml

```
@@ -0,0 +1,15 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+---
+# =============================================================================
+# Defaults for Dropbear initramfs playbook.
+# =============================================================================
+
+dropbear_initramfs__authorized_keys_file: '/etc/dropbear/initramfs/authorized_keys'
```

The quotes here are not needed, and the line is longer than 80 chars
```suggestion
dropbear_initramfs__authorized_keys_file:
  /etc/dropbear/initramfs/authorized_keys
```

This made me realise the linter is not checking yaml files, like in the other repos. I created #11 to track this.